micro-events/src/main/java/com/aol/micro/server/events/ActiveEvents.java

```diff
@@ -16,14 +17,14 @@
 
 public class ActiveEvents<T extends BaseEventInfo> {
 
-	private volatile PMap<String, T> active = HashTreePMap.empty();
+	private Map<String, T> active = new ConcurrentHashMap<>();
```

Comment:
Good spot - this code was syncrhonized, but hurriedly changed to immutable types. There are other race conditions in this class (inside finished)